### ISSUE TITLE
After changing blocks (by plugins), the blocks remain the same for Bedrock players.

### ISSUE THREAD

**Issue by BlusteryS** [GeyserMC/Geyser#1142]
**Describe the bug**
After changing blocks (WorldEdit, some minigames that change blocks, and other plugins), the blocks remain the same for Bedrock players. Only after clicking on the block, the block is replaced with the one that was replaced by the plugin. After unloading and loading a chunk with changes, the blocks apply the change.

**To Reproduce**
1. Run the command of any plugin to change blocks (for example, WorldEdit): //set 5.
2. Blocks will not change in Bedrock, but in Java they will.

**Expected Behavior**
I want the blocks to change like in the Java version.

**Server Version**
This server is running Paper version git-Paper-131 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)

**Geyser Version**
This server is running Geyser version 1.1.0 (git-master-bf238f5) (Java: 1.16.2, Bedrock: 1.16.20). I am not using Floodgate.

**Minecraft: Bedrock Edition Version**
1.16.2

**Additional Context**
https://dump.geysermc.org/2J3dRMKpBd7AXPTWu2o9MzeZ2bBRR96r

**Comment by bundabrg** [comment]
Duplicate of #399. Please follow any progress there.
